docker/spec/fixtures/helm/yaml/single-quotes.yaml

```diff
@@ -0,0 +1,4 @@
+image:
+  repository: 'nginx'
+  tag: '1.14.2'
+  registry: registry.example.com
```

Comment:
and single quote`registry.example.com` here?

Reply:
Done